tests/test_wafv2/test_server.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import unicode_literals
+import json
+import pytest
+
+import sure  # noqa
+
+import boto3
+import botocore
+from botocore.exceptions import ClientError
+import moto.server as server
+from moto import mock_wafv2
+from moto.wafv2 import GLOBAL_REGION
+from moto.wafv2.models import US_EAST_1_REGION
+from test_helper_functions import CREATE_WEB_ACL_BODY, LIST_WEB_ACL_BODY
+
+CREATE_WEB_ACL_HEADERS = {
+    "X-Amz-Target": "AWSWAF_20190729.CreateWebACL",
+    "Content-Type": "application/json",
+}
+
+
+LIST_WEB_ACL_HEADERS = {
+    "X-Amz-Target": "AWSWAF_20190729.ListWebACLs",
+    "Content-Type": "application/json",
+}
+
+
+@mock_wafv2
+def test_create_web_acl():
+    backend = server.create_backend_app("wafv2")
+    test_client = backend.test_client()
+
+    res = test_client.post("/", headers=CREATE_WEB_ACL_HEADERS, json=CREATE_WEB_ACL_BODY("John", "REGIONAL"))
+    assert res.status_code == 200
+
+    web_acl = res.json["Summary"]
+    assert web_acl.get("Name") == "John"
+    assert web_acl.get("ARN").startswith("arn:aws:wafv2:us-east-1:123456789012:regional/webacl/John/")
```

Comment:
The account ID should be parameterized in tests as well (just in case it changes later on):
from moto.core import ACCOUNT_ID